Detectors/Passive/src/Pipe.cxx

```diff
@@ -1509,9 +1637,9 @@ void Pipe::ConstructGeometry()
     dovX -= 0.0625;
     dovY += 0.075;
     char vname[20];
-    snprintf(vname, 20, "voRB242CuOvTransV%d", i);
+    sprintf(vname, "voRB242CuOvTransV%d", i);
```

Comment:
Any specific reason to do so? `sprintf` is deprecated and should be avoided. (see also CI warnings)

Reply:
Unfortunately, it seems that I didn't work on the latest version of the PIPE code... sorry for that, I should kill this PR and integrate the wings in the current version of PIPE either modify on the fly the code if it's related to few lines.